airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_creatives.json

```diff
@@ -85,7 +85,7 @@
         "link_data": {
           "properties": {
             "additional_image_index": {
-              "type": ["null", "integer"]
+              "type": ["null", "integer", "string"]
```

Comment:
so this is integer or string? Maybe better to add schema type transformation to cast this field? https://docs.airbyte.io/connector-development/cdk-python/schemas#type-transformation